CRITICAL - 2020-11-04 00:59:38 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:00:00 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:00:51 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:01:29 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:01:59 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:02:06 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:02:18 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:02:37 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:02:47 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:03:23 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:03:24 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:08:07 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:11:02 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:11:18 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:11:26 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:12:00 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:12:02 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:12:29 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(22): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:13:22 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(33): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(26): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:16:22 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(37): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(23): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:16:39 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(37): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(23): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:17:00 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `Id`, `u...', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(37): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(23): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:17:31 --> Unable to connect to the database.
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2019): CodeIgniter\Database\BaseConnection->query('SELECT `users`....', Array, false)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 D:\WORKS\Test\news\app\Models\UsersModel.php(37): CodeIgniter\Model->__call('getWhere', Array)
#4 D:\WORKS\Test\news\app\Controllers\Signup.php(23): App\Models\UsersModel->getUserByName('tunv')
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-11-04 01:19:28 --> Unknown column 'fullname' in 'field list'
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `us...')
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\WORKS\Test\news\app\Controllers\Signup.php(37): CodeIgniter\Model->insert(Array)
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#8 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-11-04 01:21:32 --> Unknown column 'phone' in 'field list'
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `us...')
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\WORKS\Test\news\app\Controllers\Signup.php(39): CodeIgniter\Model->insert(Array)
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#8 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-11-04 01:21:44 --> Unknown column 'phone' in 'field list'
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `us...')
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\WORKS\Test\news\app\Controllers\Signup.php(39): CodeIgniter\Model->insert(Array)
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Signup->submitsignup()
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Signup))
#8 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-11-04 01:38:43 --> Too few arguments to function CodeIgniter\HTTP\RedirectResponse::to(), 0 passed in D:\WORKS\Test\news\app\Controllers\Login.php on line 8 and at least 1 expected
#0 D:\WORKS\Test\news\app\Controllers\Login.php(8): CodeIgniter\HTTP\RedirectResponse->to()
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Login->index()
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-04 02:03:57 --> Invalid file: login.php
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('login', Array, true)
#2 D:\WORKS\Test\news\app\Controllers\Home.php(11): view('login')
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->index()
#4 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2020-11-04 04:45:36 --> Invalid file: home_js.php
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home_js.php')
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('home_js', Array, true)
#2 D:\WORKS\Test\news\app\Views\layout.php(3991): view('home_js', Array)
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\WORKS\\Test\\n...')
#4 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('layout', Array, true)
#5 D:\WORKS\Test\news\app\Controllers\Home.php(9): view('layout', Array)
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->index()
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-11-04 04:45:36 --> Invalid file: home_js.php
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home_js.php')
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('home_js', Array, true)
#2 D:\WORKS\Test\news\app\Views\layout.php(3991): view('home_js', Array)
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\WORKS\\Test\\n...')
#4 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('layout', Array, true)
#5 D:\WORKS\Test\news\app\Controllers\Home.php(9): view('layout', Array)
#6 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Home->index()
#7 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-11-04 21:21:20 --> Array to string conversion
#0 D:\WORKS\Test\news\app\Controllers\Users.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'D:\\WORKS\\Test\\n...', 18, Array)
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Users->loaddata()
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-04 21:21:37 --> Array to string conversion
#0 D:\WORKS\Test\news\app\Controllers\Users.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'D:\\WORKS\\Test\\n...', 18, Array)
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Users->loaddata()
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-04 21:22:28 --> Array to string conversion
#0 D:\WORKS\Test\news\app\Controllers\Users.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'D:\\WORKS\\Test\\n...', 18, Array)
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Users->loaddata()
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-04 21:23:03 --> Array to string conversion
#0 D:\WORKS\Test\news\app\Controllers\Users.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'D:\\WORKS\\Test\\n...', 18, Array)
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Users->loaddata()
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-04 21:24:14 --> Array to string conversion
#0 D:\WORKS\Test\news\app\Controllers\Users.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'D:\\WORKS\\Test\\n...', 18, Array)
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Users->loaddata()
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-04 21:24:26 --> Array to string conversion
#0 D:\WORKS\Test\news\app\Controllers\Users.php(18): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'D:\\WORKS\\Test\\n...', 18, Array)
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Users->loaddata()
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-04 21:40:44 --> Too few arguments to function CodeIgniter\Database\BaseBuilder::where(), 0 passed in D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php on line 1818 and at least 1 expected
#0 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->where()
#1 D:\WORKS\Test\news\app\Controllers\Users.php(19): CodeIgniter\Model->__call('where', Array)
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Users->loaddata()
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-11-04 21:43:05 --> Undefined index: sdt
#0 D:\WORKS\Test\news\app\Controllers\Users.php(31): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'D:\\WORKS\\Test\\n...', 31, Array)
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Users->loaddata()
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-04 21:43:12 --> Undefined index: sdt
#0 D:\WORKS\Test\news\app\Controllers\Users.php(27): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'D:\\WORKS\\Test\\n...', 27, Array)
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Users->loaddata()
#2 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#3 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-11-04 22:50:27 --> file_get_contents(): read of 8192 bytes failed with errno=13 Permission denied
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(8, 'file_get_conten...', 'D:\\WORKS\\Test\\n...', 455, Array)
#1 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Debug\Toolbar.php(455): file_get_contents('D:\\WORKS\\Test\\n...')
#2 D:\WORKS\Test\news\app\Config\Events.php(50): CodeIgniter\Debug\Toolbar->respond()
#3 [internal function]: CodeIgniter\Events\Events::Config\{closure}()
#4 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\Events\Events.php(188): call_user_func(Object(Closure))
#5 D:\WORKS\Test\news\vendor\codeigniter4\framework\system\CodeIgniter.php(293): CodeIgniter\Events\Events::trigger('pre_system')
#6 D:\WORKS\Test\news\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
